.devcontainer/Dockerfile

```diff
@@ -6,6 +6,9 @@ USER root
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
 USER $USER
 
+# copy files needed for version metadata
+COPY .git .git
```

Comment:
On second thought (and after some testing), let's remove this -- I removed it locally and tried rebuilding the devcontainer after the appcontainer, and got an error:

```log
> [dev 8/8] RUN git init . &&     pre-commit install-hooks &&     rm -rf .git:
0.101 error: could not lock config file /calitp/app/.git/config: Permission denied
0.101 fatal: could not set 'core.repositoryformatversion' to '0'
```

This is a problem with the last few lines of this file, which should also be removed -- since the `.git` directory now exists, we can't call `git init` and do the temporary pre-commit install.

```
# install pre-commit environments in throwaway Git repository
# https://stackoverflow.com/a/68758943
COPY .pre-commit-config.yaml .
RUN git init . && \
    pre-commit install-hooks && \
    rm -rf .git
```

Even trying to install the hooks without `git init` fails due to some Docker layers weirdness... in any case, this matches how we have it in Server.

Reply:
In fact, take a look at what calitp.org does in `postAttach`, I think we should apply this here and in Server: https://github.com/cal-itp/calitp.org/blob/main/.devcontainer/postAttach.sh

Reply:
Yep, I verified the behavior you described and made the suggested changes. The dev container loaded without a problem so we can make this change to eligibility-server as you mentioned.